### DISCUSSION TITLE
how to configure the HMI

### DISCUSSION THREAD

**Discussion by BlueScreenTT** [arendst/Tasmota#14753] [Q&A] [answered]
anyone got any links to resources for using NSpanel after a tasmota update.
i mean how to configure the screen, changing icons and actualy getting it to work

The nspanel is kind of useles with the tasmota firmware it it is not possible to configure the device (HMI)

BR
Thomas

**Comment by sfromis** [comment, marked as answer]
Did you see https://templates.blakadder.com/sonoff_NSPanel.html **and** follow the links to further articles?

**Comment by BlueScreenTT** [comment]
Didnt see any links.   Have to take a 2'nd look :-)
Thank you for the quick answer

BR Thomas

Den sön 6 feb. 2022 23:58sfromis ***@***.***> skrev:

**Reply by chriskmn** [reply]
Is there any chance to get the „ USE_THERMOSTAT“ option activated in the NSPanel build ?
